sparse/src/KokkosSparse_CrsMatrix.hpp

```diff
@@ -42,7 +42,7 @@
 //@HEADER
 */
 
-/// \file Kokkos_Sparse_CrsMatrix.hpp
+/// \file KokkosSparse_CrsMatrix.hpp
```

Comment:
I don't get these changes. Why did we modify CrsMatrix so heavily for writing a utility for conversion?